### PR TITLE
Create Mock for PostCodeLib

### PR DESCRIPTION
**Pull request by TsunFeng**
## Description

Create Mock for PostCodeLib

For details on how to complete these options and their meaning refer to [CONTRIBUTING.md](https://github.com/microsoft/mu/blob/HEAD/CONTRIBUTING.md).

- [ ] Impacts functionality?
- [ ] Impacts security?
- [ ] Breaking change?
- [ ] Includes tests?
- [ ] Includes documentation?
- [x] Backport to release branch?

## How This Was Tested

Unit tests component can call these mock functions success

## Integration Instructions

N/A

### CODE REVIEW

File: MdePkg/Test/Mock/Include/GoogleTest/Library/MockPostCodeLib.h [hunk 1]
```diff
@@ -0,0 +1,54 @@
+/** @file MockPostCodeLib.h
+  Google Test mocks for PostCodeLib
```
Comment (VivianNK) [inline MdePkg/Test/Mock/Include/GoogleTest/Library/MockPostCodeLib.h:2]: Please add the copyright from the header file of the library being mocked:
> Copyright (c) 2006 - 2018, Intel Corporation. All rights reserved

**Review by apop5 — APPROVED** [review]

File: MdePkg/Test/MdePkgHostTest.dsc [hunk 2]
```diff
@@ -56,6 +56,7 @@
   MdePkg/Test/Mock/Library/GoogleTest/MockMmServicesTableLib/MockMmServicesTableLib.inf
   MdePkg/Test/Mock/Library/GoogleTest/MockUefiRuntimeLib/MockUefiRuntimeLib.inf
   MdePkg/Test/Mock/Library/GoogleTest/MockMemoryAllocationLib/MockMemoryAllocationLib.inf
+  MdePkg/Test/Mock/Library/GoogleTest/MockPostCodeLib/MockPostCodeLib.inf
```
Comment (VivianNK) [inline MdePkg/Test/MdePkgHostTest.dsc]: Please move this line to be included in the MU_CHANGE tag below

**Review by VivianNK — APPROVED** [review]

File: MdePkg/Test/MdePkgHostTest.dsc (hunk 2 above)
Comment (TsunFeng) [reply MdePkg/Test/MdePkgHostTest.dsc]: Done

**Comment by TsunFeng** [comment]
Hi @os-d,
I would appreciate it if you could review this PR.

thanks,
TF

**Pull request merged**